README.md

```diff
@@ -1,31 +1,179 @@
-![](https://img.shields.io/badge/Microverse-blueviolet)
-# Linters Test
-> This project is to test linters
-## Built With
-- HTML and CSS
+<a name="readme-top"></a>
 
-## Getting Started
+<div align="center">
+
+  <img src="murple_logo.png" alt="logo" width="140"  height="auto" />
```

Comment:
- [ ] Kindly fix your logo here or remove it for proper documentation.

  ![ffjfj](https://user-images.githubusercontent.com/36542986/207307849-cbe49f8a-2cdb-4909-9d7a-04866daa5418.jpg)